internal/component/pyroscope/scrape/scrape_loop.go

```diff
@@ -228,6 +241,9 @@ func (t *scrapeLoop) scrape() {
 	)
 	defer cancel()
 
+	scrapeCtx, span := t.tracer.Tracer("").Start(scrapeCtx, "scrape")
```

Comment:
Naming spans is the most difficult thing. This is the only span I create explicitly in this PR, so there's not much naming convention to do, but it would be good to have one.